graph-traversal/graphtrav.py

```diff
@@ -0,0 +1,42 @@
+# __ coding utf-8 __
```

Comment:
These need to be methods on your Graph object, not separate functions. Check the guidelines of the assignment.